src/tvhlog.c

```diff
@@ -36,13 +36,11 @@
 #include "tcp.h"
 #include "webui/webui.h"
 
-#define TVHLOG_BITARRAY ((LS_LAST + (BITS_PER_LONG - 1)) / BITS_PER_LONG)
-
 int                      tvhlog_run;
 int                      tvhlog_level;
 int                      tvhlog_options;
 char                    *tvhlog_path;
-bitops_ulong_t           tvhlog_debug[TVHLOG_BITARRAY];
+bitops_ulong_t           tvhlog_debug[TVHLOG_BITARRAY];  //TVHLOG_BITARRAY is defined in bitops.h
```

Comment:
Now this text is in the middle of the 2 usages, I suggest moving it up a line.

Reply:
Done.